packages/dashboard/package.json

```diff
@@ -68,12 +69,11 @@
     "postcss-import": "^14.0.0",
     "postcss-url": "^10.1.3",
     "react": "^17.0.2",
-    "react-dnd-test-backend": "^16.0.1",
-    "react-dnd-test-utils": "^16.0.1",
+    "react-dnd-test-backend": "^15.1.2",
```

Comment:
why do we have to downgrade this?

Reply:
according to https://github.com/react-dnd/react-dnd/issues/3429, react-dnd 16 have issue with react v17. The latest good version is react-dnd@15.1.2